app/components/Contexts.tsx

```diff
@@ -5,11 +5,16 @@ import type { PropsWithChildren } from "react";
 import AmplitudeProvider from "@/providers/AmplitudeProvider";
 import QueryProvider from "@/providers/QueryProvider";
 import { PosthogProvider } from "@/providers/PosthogProvider";
+import { QueryClient, QueryClientProvider } from "react-query";
+
+const queryClient = new QueryClient();
 
 const Contexts = ({ children }: PropsWithChildren) => (
   <PosthogProvider>
     <AmplitudeProvider>
-      <QueryProvider>{children}</QueryProvider>
+      <QueryClientProvider client={queryClient}>
+        <QueryProvider>{children}</QueryProvider>
+      </QueryClientProvider>
```

Comment:
This is still adding an unnecessary extra QueryClientProvider (that's what QueryProvider is doing already)